packages/access-api/test/ucan.test.js

```diff
@@ -144,6 +144,31 @@ describe('ucan', function () {
     t.deepEqual(rsp, ['test pass'])
   })
 
+  test('should support ucan invoking to a did:web aud', async function () {
+    const serviceDidWeb = 'did:web:web3.storage'
+    const { mf, issuer, service } = await context({
+      environment: {
+        ...process.env,
+        PRIVATE_KEY:
+          'MgCYWjE6vp0cn3amPan2xPO+f6EZ3I+KwuN1w2vx57vpJ9O0Bn4ci4jn8itwc121ujm7lDHkCW24LuKfZwIdmsifVysY=',
```

Comment:
why do you need a new key here ?

Reply:
I don't. Good catch. 🙏  https://github.com/web3-storage/w3protocol/pull/275/commits/d20433140493ecf5408c0e6edb49ce654ec01f12